packages/gatsby-plugin-sharp/src/scheduler.js

```diff
@@ -92,31 +82,34 @@ const scheduleJob = async (
       { name: `gatsby-plugin-sharp` }
     )
 
-    q.push(cb => {
-      runJobs(
-        jobId,
-        getFileKey(job.inputPath),
-        boundActionCreators,
-        pluginOptions,
-        reportStatus,
-        cb
-      )
-    })
+    runJobs(
+      jobId,
+      getFileKey(job.inputPath),
+      boundActionCreators,
+      pluginOptions,
+      jobs => {
+        jobsFinished++
+
+        // only show progress on build
+        jobs.forEach(() => {
+          if (reportStatus) {
+            bar.tick()
+          }
+        })
+
+        if (totalJobs === jobsFinished) {
+          bar.done()
+          totalJobs = 0
```

Comment:
This is not regression, but shouldn't we also set `jobsFinished` to `0` as well here if we are resetting that state?